lib/nodejs/worker.ex

```diff
@@ -50,38 +50,67 @@ defmodule NodeJS.Worker do
     port =
       Port.open(
         {:spawn_executable, node},
-        line: @read_chunk_size,
-        env: [
-          {~c"NODE_PATH", node_path(module_path)},
-          {~c"WRITE_CHUNK_SIZE", String.to_charlist("#{@read_chunk_size}")}
-        ],
-        args: [node_service_path()]
+        [
+          {:line, @read_chunk_size},
+          {:env, get_env_vars(module_path)},
+          {:args, [node_service_path()]},
+          :exit_status,
+          :use_stdio,
+          :stderr_to_stdout
```

Comment:
It seems like we were already using `stdio` by default, and that the `:stderr_to_stdout` is the really impactful change here, is that right?

Reply:
this is the result of multiple iterations, I tried again this morning and indeed, we can remove the :use_stdio